WORKING_GROUPS.md

```diff
@@ -0,0 +1,296 @@
+# io.js Working Groups
+
+io.js Working Groups are autonomous projects created by the TC.
+
+Working Groups can be formed at any time but must be ratified by the TC.
+Once formed the work defined in the Working Group charter is the
+responsibility of the WG rather than the TC.
+
+It is important that Working Groups are not formed pre-maturely. Working
+Groups are not formed to *begin* a set of tasks but instead are formed
+once that work is already underway and the contributors
+think it would benefit from being done as an autonomous project.
+
+If the work defined in a Working Group charter is completed the Working
+Group should be dissolved and the responsibility for governance absorbed
+back in to the TC.
+
+## Current Working Groups
+
+### Website
+
+The website working group's purpose is to build and maintain a public
+website for the `io.js` project.
+
+Its responsibilities are:
+* Develop and maintain a build and automation system for `iojs.org`.
+* Ensure the site is regularly updated with changes made to `io.js` like
+releases and features.
+* Foster and enable a community of translators.
+
+The current members can be found in their
+[README](https://github.com/iojs/website#current-project-team-members).
+
+### Streams
+
+The streams working group's purpose is to improve the existing Stream
+implementation, in accordance with the communities needs and feedback.
+
+Its responsibilities are:
+* Produce a living `Stream` standard.
+* Create a reference implementation as `readable-stream`.
+* Recommend versions of `readable-stream` to be included in `io.js`
+* Collaborate with the WHATWG Stream standard to ensure an optimal level
+of compatibility between the two standards.
+
+Initial members are:
+* @chrisdickinson
+* @isaacs
+* @rvagg
+* @TooTallNate
+* @Raynos
+* @calvinmetcalf
+* @substack
+* @sonewman
+* @mafintosh
+* @timgestson
+* @deoxxa
+* @maxogden
+* @feross
+* @mafintosh
+* @calvinmetcalf
+* @sonewman
+* @domenic
+* @timgestson
```

Comment:
Ya, the only reason this list is here is because the readable-stream README doesn't have one :) I pulled this from a GH issue.